ci/tasks/check-if-usn-is-in-github.sh

```
@@ -0,0 +1,35 @@
+touch "${REPO_PARENT}/joined-usns"
+cp "${REPO_PARENT}/unfound-usns/usns.json" "${REPO_PARENT}/joined-usns"
+cat "${REPO_PARENT}/usn/usn.json" >> "${REPO_PARENT}/joined-usns"
+
+# turn the unfound usns into an array
+mapfile -t UNFOUND_USNS < "${REPO_PARENT}/joined-usns"
+
+# check if each of the unfound usns is in github
+touch "${REPO_PARENT}/found-usns/usns.json"
+touch "${REPO_PARENT}/updated-unfound-usns/usns.json"
+for usn in "${UNFOUND_USNS[@]}"; do
+    id=$(echo $usn | jq -r '.url' | cut -d '/' -f6 | cut -d '-' -f2,3)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)